src/commands/init.ts

```diff
@@ -101,3 +137,13 @@ export async function init(type?: ProjectType) {
 
   vscode.window.showInformationMessage('Project created successfully!')
 }
+
+export async function initSdk7() {
+  const child = bin('@dcl/sdk', 'sdk-commands', ['init', `--skip-install`])
+
+  await child.wait()
+
+  await npmInstall()
```

Comment:
is this correct? skipping install on `init` but installing after?

Reply:
yes, but actually I realized that the `init` command from `@dcl/sdk` does not do an npm install automatically, so I can remove it. But on the other CLI we do skip it because what it does is it runs `npm install`, and that fails on a computer that has not installed npm globally. So we skip it, and we do the npm install ourselves (the editor has npm as its own dependency so it does not require it to be installed globally to work).

Reply:
yeah, `init` doesn't install deps but `build` & `start` do. Nice